docs/sources/flow/reference/components/otelcol.receiver.vcenter.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.receiver.vcenter/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.receiver.vcenter/
+title: otelcol.receiver.vcenter
+description: Learn about otelcol.receiver.vcenter
+---
+
+# otelcol.receiver.vcenter
+
+`otelcol.receiver.vcenter` accepts metrics from a 
+vCenter or ESXi host running VMware vSphere APIs and
+forwards it to other `otelcol.*` components.
+
+> **NOTE**: `otelcol.receiver.vcenter` is a wrapper over the upstream
+> OpenTelemetry Collector `vcenter` receiver from the `otelcol-contrib`
+> distribution. Bug reports or feature requests will be redirected to the
+> upstream repository, if necessary.
+
+Multiple `otelcol.receiver.vcenter` components can be specified by giving them
+different labels.
+
+Make sure that you fulfill the [prerequisites](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/vcenterreceiver/README.md#prerequisites) before using this component.
```

Comment:
There are a couple of issues with linking to the OTEL docs like this:
* This link goes to the main branch. It'd ideally go to the version of OTEL which we use - see the otelcol.processor.transform for example URLs.
* There are various things listed in the page, including Collector yampl config examples and Collector feature gates. This could be confusing to the Agent users. 

I'd prefer if we just copy over the relevant information:

> This receiver has been built to support ESXi and vCenter versions:
> 
> * 7.5
> * 7.0
> * 6.7
> 
> A “Read Only” user assigned to a vSphere with permissions to the vCenter server, cluster and all subsequent resources being monitored must be specified in order for the receiver to retrieve information about them.